apps/remix-ide/src/app/files/foundry-handle.js

```diff
@@ -15,4 +15,12 @@ export class FoundryHandle extends WebsocketPlugin {
   constructor () {
     super(profile)
   }
+
```

Comment:
might need to do the same for all the other WebsocketPlugin plugin (slither, remixd, ...)

Reply:
ok